boards/arm/mimx8mp_evk/mimx8mp_evk_itcm.dts

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) 2021, Laird Connectivity
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+
+#include <nxp/nxp_imx8ml_m7.dtsi>
+
+/ {
+	model = "NXP i.MX8M Plus EVK board";
+	compatible = "nxp,mimx8mp_evk";
+
+	aliases {
+		uart-4 = &uart4;
```

Comment:
These types of aliases are generally not needed in new boards. They occur in old boards as a holdover from the days before we had things like DT_NODELABEL. Do you really need this?

Reply:
Removed!